code/contracts/ONEWallet.sol

```diff
@@ -38,16 +39,16 @@ contract ONEWallet {
         height = height_;
         interval = interval_;
         t0 = t0_;
-        lifespan_ = lifespan_;
+        lifespan_ = lifespan_; // IH: what are the expected values?
```

Comment:
It would be (validity time in seconds / interval) where interval should be 30 for standard uses. I intend to use a default value of 1 year for validity time in the client.

Reply:
One year is more than enough for the basic wallet. The feedback I got is to make it even shorter, since we are going to upgrade the wallet pretty soon anyway. Later we can provide a way to refresh wallet lifespan after the user activates another authentication factor.